tests/karma/all.conf.js

```diff
@@ -50,6 +50,7 @@ module.exports = function (config) {
 			{pattern: "libs/**/*.js", included: false, served: true, watch: true},
 			{pattern: "dist/**/*", included: false, served: true, watch: true},
 			{pattern: "libs/path-to-regexp.js", included: false, served: true},
+			{pattern: "libs/BezierCurve.js", included: false, served: true},
 			"tests/karma/testPaths.js",
```

Comment:
I think that file`{pattern: "some-script.js", included: false, served: true},`  is also needed.

Reply:
right, done